### PR TITLE
feat: add flair indexing

### PR DESCRIPTION
**Pull request by kasrakhosravi**
**What changed? Why?**
Adding [Flair](https://flair.dev)  as a data indexing solution on Base

**Notes to reviewers**
Screenshot 2023-10-24 at 15 39 16

**How has it been tested?**
yes

### CODE REVIEW

**Review by zencephalon — CHANGES_REQUESTED** [review]
@kasrakhosravi Thanks for your contribution! Would U mind removing the large image in order to fit in better with the other entries? I don't want to create a dynamic where everyone feels like they've got to make a huge banner in order to grab eyeballs.

**Review by zencephalon — APPROVED** [review]

File: apps/base-docs/docs/tools/data-indexers.md [hunk 1]
```diff
@@ -56,3 +56,16 @@ To get started, visit the [documentation](https://thegraph.com/docs/en/) or see
 - [Base Goerli](https://thegraph.com/docs/en/#supported-networks) (Testnet)
 
 ---
+
+## Flair
+
+[Flair](https://flair.dev) is a real-time and historical custom data indexing for any evm chain.
```
Comment (rcbgr) [inline apps/base-docs/docs/tools/data-indexers.md:62]: uppercase EVM

**Review by rcbgr — CHANGES_REQUESTED** [review]

**Comment by cb-heimdall** [comment]
Review Error for rcbgr @ 2023-10-24 17:33:39 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

File: apps/base-docs/docs/tools/data-indexers.md (hunk 1 above)
Comment (rcbgr) [inline apps/base-docs/docs/tools/data-indexers.md:62]: Uppercase EVM

**Review by rcbgr — CHANGES_REQUESTED** [review]

File: apps/base-docs/docs/tools/data-indexers.md (hunk 1 above)
Comment (kasrakhosravi) [reply apps/base-docs/docs/tools/data-indexers.md]: fixed

**Review by roberto-bayardo — APPROVED** [review]

**Review by zencephalon — APPROVED** [review]

**Pull request merged**